sdk/servicebus/azservicebus/Makefile

```diff
@@ -0,0 +1,101 @@
+PACKAGE  = github.com/Azure/azure-service-bus-go
```

Comment:
I don't think we need this file anymore

Reply:
I'll delete it - this repo uses a combination of things that aren't "Azure standard" so we'll have to fix those first.

For instance, this Makefile handles things like running terraform, etc... So there's still work that has to happen to get us formally integrated into CI.